src/area-chart/area-chart-stacked.component.ts

```diff
@@ -356,8 +361,13 @@ export class AreaChartStackedComponent extends BaseChartComponent {
       domain.push(sum);
     }
 
-    const min = Math.min(0, ...domain);
-    const max = Math.max(this.yAxisMinScale, ...domain);
+    const min = this.yScaleMin
+      ? this.yScaleMin
+      : Math.min(0, ...domain);
+
+    const max = this.yScaleMax
```

Comment:
Wrong value check, I think
this.yScaleMax !== undefined && this.yScaleMax !== null
should be better ;)

if you use lodash, !_.isNil(this.yScaleMax)